src/SDKs/DataMigration/DataMigration.Tests/DataMigration.Tests.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>DataMigration.Tests</PackageId>
+    <VersionPrefix>1.0.0-preview</VersionPrefix>
+    <AssemblyName>DataMigration.Tests</AssemblyName>
+    <Description>DataMigration.Tests Class Library</Description>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.KeyVault" Version="2.3.2" />
+    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.6.0-preview" />
+    <PackageReference Update="Microsoft.NET.Test.Sdk" Version="15.3.0" />
```

Comment:
@hitenjava Microsoft.NET.Test.Sdk and Rest.ClientRuntime.Azure.TestFramework references are implicitly added to your project, so you don't have to explicitlty add them.
When we change these packages or upgrade them, your project will be automatically upgraded at the same time.
Having such explicit references inhibit us to upgrade repo wide such changes.

Reply:
Fixed.